internal/services/containers/kubernetes_cluster_node_pool_resource_test.go

```diff
@@ -2442,3 +2457,43 @@ resource "azurerm_kubernetes_cluster_node_pool" "test" {
 }
 `, data.Locations.Primary, data.RandomInteger)
 }
+
+func (KubernetesClusterNodePoolResource) nodeIPTags(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-aks-%[2]d"
+  location = "%[1]s"
+}
+
+resource "azurerm_kubernetes_cluster" "test" {
+  name                = "acctestaks%[2]d"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+  dns_prefix          = "acctestaks%[2]d"
+  default_node_pool {
+    name       = "default"
+    node_count = 1
+    vm_size    = "Standard_D2s_v3"
+  }
+  identity {
+    type = "SystemAssigned"
+  }
+}
+
+resource "azurerm_kubernetes_cluster_node_pool" "test" {
+  name                  = "internal"
+  kubernetes_cluster_id = azurerm_kubernetes_cluster.test.id
+  vm_size               = "Standard_D2s_v3"
+  enable_node_public_ip = true
+  network_profile {
+    node_public_ip_tags = {
+      RoutingPreference = "Internet"
```

Comment:
Can we set multiple tags here

Reply:
Hi @stephybun , I've confirmed with the feature owner, that now it only supports one ip tag, but will support multiple in the future.